drivers/sdhc/brcm_bcm2712_sdhci.c

```diff
@@ -453,7 +463,18 @@ static int bcm2712_sdhci_init(const struct device *dev)
 	return 0;
 }
 
+#if defined(CONFIG_BRCM_BCM2712_XFER_ADMA2)
+#define BRCM_BCM2712_ADMA_DESCS(inst)                                                              \
```

Comment:
maybe it worth to use _ADMA2_ in defines to be consistent.

Reply:
It's in PR this one is based on.